jena-fuseki2/jena-fuseki-main/src/main/java/org/apache/jena/fuseki/main/cmds/FusekiMain.java

```diff
@@ -282,21 +282,26 @@ protected void processModulesAndArgs() {
         }
 
         if ( contains(argFile) ) {
-            String filename = getValue(argFile);
-            String pathname = filename;
-            if ( filename.startsWith("file:") )
-                pathname = filename.substring("file:".length());
-
-            serverConfig.datasetDescription = "file:"+filename;
-            if ( !FileOps.exists(pathname) )
-                throw new CmdException("File not found: " + filename);
-            serverConfig.dsg = DatasetGraphFactory.createTxnMem();
-
-            // INITIAL DATA.
-            Lang language = RDFLanguages.filenameToLang(filename);
-            if ( language == null )
-                throw new CmdException("Can't guess language for file: " + filename);
-            Txn.executeWrite(serverConfig.dsg,  ()->RDFDataMgr.read(serverConfig.dsg, filename));
+            List<String> filenames = getValues(argFile);
+            String desc = filenames.size() == 1
+                ? "file: "+filenames.get(1)
```

Comment:
Isn't the parameter the index of the element? Or is it the number of elements to pop/shift?